lua/nvcommunity/tools/presence/init.lua

```diff
@@ -0,0 +1,10 @@
+---@type NvPluginSpec
+local spec = {
+  {
+    "andweeb/presence.nvim",
+    event = "VimEnter",
```

Comment:
Wouldnt be better be `VeryLazy`?

Reply:
It works with both, but with `VeryLazy`, when you start nvim, it doesn't display until you open a buffer. That's why I used `VimEnter`

Reply:
that's now how `VeryLazy` is supposed to work, im pretty sure

Reply:
`VeryLazy`, despite the name, is just similar to this

```lua
-- some callback_fn()
vim.api.nvim_create_autocmd("VimEnter", {
  callback = function()
   vim.schedule(callback_fn)
  end
})
```